static/js/common.js

```diff
@@ -62,6 +62,22 @@ exports.password_quality = function (password, bar, password_field) {
     return acceptable;
 };
 
+exports.password_warning = function (password, password_field) {
+    if (typeof zxcvbn === 'undefined') {
+        return undefined;
+    }
+
+    var min_password_length = 6;
+    if (password_field) {
+        min_password_length = password_field.data('minLength') || min_password_length;
```

Comment:
I understand settings better now. this will always be defined, so we don't need an `|| min_password_length` here. Same with the zxcvbn one.